### ISSUE TITLE
PrimeReact scroll, Virtual Scrolling

### ISSUE THREAD

**Issue by jtlindsey** [primefaces/primereact#116]
I'm trying out primereact scroll feature. According to the [docs](https://www.primefaces.org/primereact/#/datatable/scroll) 

> Virtual Scrolling mode is available to deal with large datasets by loading data on demand during scrolling..

In my code i copied the code you used for get cars large and place my serverurl. On mount, i get data from my server as expected, but I don't see how to tell the server how much data to fetch at one time or how to fetch more data based on percent scrolled to bottom, etc.

```
  getCarsLarge(data={filter:{}}) {
    console.log('******call carservice')
    return axios.post('/getstuff', data).then((res) => {
      return res.data.data
    });
  }
```

How do you get the following feature to work?

> Virtual Scrolling mode is available to deal with large datasets by loading data on demand during scrolling..

**Comment by cagataycivici** [comment]
There is an error in the docs, virtual scrolling is scheduled for upcoming 1.0.2, it is not available yet but will be later this week.

**Comment by cagataycivici** [comment]
Tracked at #79

**Comment by cagataycivici** [comment]
Implemented for 1.1.0.

**Comment by jtlindsey** [comment]
Thank you
